ui/public/brand/assets/branding-light.svg

```diff
@@ -0,0 +1,50 @@
+<svg
```

Comment:
> I think we do not need to add a new `brand[light/dark].svg` because the difference is small. [...] Thus, we don't need the `logoPath`

True, we can add styling directly to the SVG. We could also handle it in JS, FWIW. In the meantime, having a single path to set a logo, be it a default Scality-provided theme or a custom one, seems a reasonable first iteration. I think we can stick to 2 SVGs for now and open an improvement ticket to handle it some time later.

> for the "custom" theme, we definitely need the FULL `logoPath`, since the `logo` could be mounted somewhere else than `process.env.PUBLIC_URL`. For now, we don't need to support this yet. @gdemonet any thoughts?

The only good reason for the logo not being exposed under the same URL as the UI itself is to use an absolute URL (say, from a CDN). IMHO, it's not urgent.
For now, changing a logo would mean:
- adding it as binary data to the `metalk8s-ui/ui-branding` _ConfigMap_
- referencing it from the root of the application (which is what `PUBLIC_URL` represents) in the `theme.json` `"logoPath"`, e.g. `"/brand/custom_logo.svg"`

I'm however now doubting about the _ConfigMap_ approach. It's sufficient for simple theming through `theme.json` values, but as soon as we start adding multiple files, it becomes cumbersome to say the least. As an example, adding sub-directories under `brand` from this single _ConfigMap_ wouldn't be feasible.
Also, the whole idea of sharing this theming configuration between UIs in the cluster means we need a more flexible approach than this single file.

Some solutions come to mind:
- have _another_ static container for branding assets, and let all UIs expose it with their own _Ingress_
    - (+) platform operators can simply edit files on the Bootstrap (including `theme.json`) and the changes are exposed through all the UIs
    - (-) need to backup, propagate to other Bootstrap nodes (maybe even Infra? along with the UI containers...)
- generate the _ConfigMap_**(s)** from static configuration and files using Salt:
    - (+) users still have a single entrypoint
    - (+) this could be used by Solutions, if they want to share theming assets, e.g. for their NavBar icon
    - (-) all UI containers need to be updated when we add a new _ConfigMap_

... :thinking: What if we try both approaches at once? Have a static `nginx`, with its Ingress for e.g. `/brand`, that exposes files from two _ConfigMap_ sources: system defaults and user configuration. As in the current UI containers, system defaults are used as fallback when user-provided values are not available. However, instead of having those stored at build-time, we will need to mount them (this is necessary to add new system defaults dynamically, e.g. for Solution UIs). User-provided configuration can be generated from Salt / Metal UI / whatever.

If you need to upgrade/downgrade Metal / a Solution, change the system defaults _ConfigMap_(s). The user-provided (or generated) _ConfigMap_(s) will be persisted.